README.md

```diff
@@ -69,30 +59,30 @@ Using `npx` you can then run the following to create an app in a chosen subdirec
 npx @backstage/create-app
 ```
 
-You will be taken through a wizard to create your app, and the output should look something like this. You can read more about this process [here](docs/getting-started/create-an-app.md)
+You will be taken through a wizard to create your app, and the output should look something like this. You can read more about this process [here](https://backstage.io/docs/getting-started/create-an-app)
 
 ### Contributing to Backstage
 
 You can read more in our [CONTRIBUTING.md](./CONTRIBUTING.md#get-started) guide, which can help you get setup with a Backstage development environment.
 
 ### Next steps
 
-Take a look at the [Getting Started](docs/getting-started/index.md) guide to learn how to set up Backstage, and how to develop on the platform.
+Take a look at the [Getting Started](https://backstage.io/docs/getting-started/index) guide to learn how to set up Backstage, and how to develop on the platform.
 
 ## Documentation
 
-- [Main documentation](docs/README.md)
-- [Service Catalog](docs/features/software-catalog/index.md)
-- [Architecture](docs/overview/architecture-terminology.md) ([Decisions](docs/architecture-decisions/index.md))
-- [Designing for Backstage](docs/dls/design.md)
-- [Storybook - UI components](http://backstage.io/storybook)
+- [Main documentation](https://backstage.io/docs/overview/what-is-backstage)
+- [Service Catalog](https://backstage.io/docs/features/software-catalog/software-catalog-overview)
+- [Architecture](https://backstage.io/docs/overview/architecture-terminology) ([Decisions](https://backstage.io/docs/architecture-decisions/adrs-overview))
```

Comment:
I am starting to wonder if ADRs should be really part of backstage.io/docs? 🤷‍♂️

Reply:
Why not?